old/onlyoffice.sh

```diff
@@ -170,20 +174,20 @@ fi
 install_certbot
 
 # Generate certs
-if le_subdomain
+if generate_cert "$SUBDOMAIN"
 then
     # Generate DHparams chifer
-    if [ ! -f "$DHPARAMS" ]
+    if [ ! -f "$DHPARAMS_SUB" ]
     then
-        openssl dhparam -dsaparam -out "$DHPARAMS" 4096
+        openssl dhparam -dsaparam -out "$DHPARAMS_SUB" 4096
     fi
     printf "%b" "${IGreen}Certs are generated!\n${Color_Off}"
     a2ensite "$SUBDOMAIN.conf"
-    restart_webserver
+    check_command systemctl reload apache2
```

Comment:
Please revert this change, we use a function to only change on one place instead of several in case we need to change method.

Reply:
Is reverted 👍